data_safe_haven/infrastructure/programs/declarative_sre.py

```diff
@@ -350,6 +350,7 @@ def __call__(self) -> None:
                 apt_proxy_server_hostname=apt_proxy_server.hostname,
                 data_collection_rule_id=monitoring.data_collection_rule_vms.id,
                 data_collection_endpoint_id=monitoring.data_collection_endpoint.id,
+                database_service_admin_password=data.password_database_service_admin,
```

Comment:
Has something weird happened with the indentation here?

Reply:
Not that I can see, am I missing something?